### ISSUE TITLE
Add support for MFA UI

### ISSUE THREAD

**Issue by bhaumik55231** [firebase/firebaseui-web#843]
Cloud Identity Platform supports MFA, but the firebase UI seems to doesn't support this and requires developing a custom UI. It would be very helpful if this could be included out of the box.

**Comment by xil222** [comment]
#782 We're currently working on designing a short-term fix for exposing a callback to handle 2fa.
A FirebaseUI built-in implementation for the 2nd factor handler is a larger feature needs to be in sync with
other the FirebaseUI repos which would be a longer process.

**Comment by bhaumik55231** [comment]
Thanks, @xil222 , we would greatly appreciate this feature as we are using firebaseui for high visibility [cancer prevention study](https://dceg.cancer.gov/research/who-we-study/cohorts/connect) at National Cancer Institute.

**Comment by bojeil-google** [comment]
Hey @bhaumik55231 can you also file a feature request for this via [official channels](https://firebase.google.com/support) so it gets more visibility?

**Comment by bhaumik55231** [comment]
Will do @bojeil-google, Thank you!!

**Comment by astyltsvig** [comment]
Any update on this?

**Comment by xil222** [comment]
@astyltsvig There're some other feature request/bugs with higher priorities, although design is almost done, have to move towards other projects for some time. I'll try to multi-task on this when have some time and be able to support this feature in H2, sry about that.

**Comment by rscotten** [comment]
@xil222 The industry that we serve just had a major hacking (allegedly by Russians and the FBI is involved) and our clients are now motivated to opt-into MFA.   We'd really appreciate if this can be given priority attention (what's more important than the security of our data?).  We look silly when we have to explain to our customers that MFA works for email and not OAuth.

**Comment by xil222** [comment]
Hi @rscotten, I am really sorry about the delay and what's happening. I agree with you that security should be the 1st concern. 
Hey @bojeil-google , aside from other high priority projects, do you think the MFA feature can be put as the 1st priority feature request at least in FirebaseUI?

**Comment by gicola** [comment]
Why is @bojeil-google quiet?

**Comment by bayareacoder** [comment]
@meena-erian Exactly. Why was this closed? MFA is a requirement now - makes this library not really usable. No word in the official Firebase docs that MFA is not supported by this. Shows a clear lack of attention by Google to this library. 
@bojeil-google @xil222

**Comment by rscotten** [comment]
@xil222 still waiting on this.  Is this on the roadmap?

**Comment by KennyGeensSkyebase** [comment]
@xil222 @bojeil-google Pinging this again since it's still unclear. Is this being developed?

**Comment by dackers86** [comment]
Thank you for taking the time to submit this feature request.

We are happy to let you know that a resolution is already planned and on the roadmap for our upcoming new version. 

We are actively working on the new SDK and will be sure to update this request once we have a public beta or a release candidate that includes this functionality. Please watch the main repository for the latest release.

In the meantime, we will keep this issue open for tracking.

**Comment by russellwheatley** [comment]
v7 beta has been released with MFA support. We no longer support v6, if you find an issue on v7, feel free to file a new issue.
